static/panes/editor.js

```diff
@@ -351,6 +351,8 @@ Editor.prototype.onDidChangeCursorPosition = function (e) {
 };
 
 Editor.prototype.onDidFocusEditorText = function () {
+    var position = this.editor.getPosition();
+    this.currentCursorPosition.text('(' + position.lineNumber + ', ' + position.column + ')');
```

Comment:
```suggestion
    if (position) {
        this.currentCursorPosition.text('(' + position.lineNumber + ', ' + position.column + ')');
    }
```

as getPosition() can return null